src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Shape[] shapes = new Shape[6];
+        ShapeSupplier generatedShape = new ShapeSupplier();
+
+        for (int i = 0; i < shapes.length / 2; i++) {
+            shapes[i] = generatedShape.getRandomShape();
+            shapes[i].draw();
+        }
+        for (int i = shapes.length / 2; i < shapes.length; i++) {
+            shapes[i] = generatedShape.getDefaultShape();
```

Comment:
The `getDefaultShape` method should return a white circle with a radius of 10, as specified in the task description. Ensure this is correctly implemented in the `ShapeSupplier` class .